gnovm/tests/integ/typecheck-missing-return/gno.mod

```diff
@@ -0,0 +1 @@
+module gno.land/p/integ/valid
```

Comment:
I dislike this. We are checking Gno language, the same one we will use for a blockchain-less experience. Please make the `gno.mod` file optional for this check.

Reply:
hm?

I don't quite understand here; either we strongly incentivise users to use gno.mod (and suggest it in gno lint) or we don't

Here the gno.mod exists just because `gno lint` will otherwise warn about a missing gno.mod

I can remove the check, of course, but I'd want to make sure of what our intention is here